### DISCUSSION TITLE
Hasura Cloud release cadence

### DISCUSSION THREAD

**Discussion by drl-max** [hasura/graphql-engine#8141] [General] [answered]
Greetings! I've been working on a project at my company that utilizes Hasura Cloud and ran into issue #4796. Luckily for me, I discovered that issue thread after it had been resolved and only days before the recent `2.2.0` release that included the improvement.  However, the Cloud version still appears to be from the `2.0.10` release and I couldn't for the life of me find any information regarding how long it would take for these newer releases to land on the Cloud. 

Is there any information that can be shared about when we might see these latest releases make it to the Cloud?

PS. I realize that nothing prevents us from using the community releases instead, if having the latest changes is a must. Understandably, the organization wouldn't go for that so, I'm just curious.

Thanks!

**Comment by vaishnavigvs** [comment, marked as answer]
Hey! Thanks for reaching out. We're currently working on bringing the 2.2 OSS release features onto Hasura Cloud. We're aiming to have Cloud upgrades follow the OSS release by 2 weeks or less going forward. As for 2.2 release on Cloud, we're looking at enabling it this month. Please keep an eye on the monthly newsletter for cloud release updates.

**Reply by drl-max** [reply]
That's fantastic news! Thanks for letting me know.  

I look forward to the upcoming release :D
